src/homework/hom20/Factory/FurnitureFactory.java

```diff
@@ -0,0 +1,11 @@
+package homework.hom20.Factory;
+public class FurnitureFactory {
```

Comment:
It's a not an AbstractFactory. You need to make it as abstract class or interface and create a couple of factory imlementations

Reply:
corrected